server.R

```diff
@@ -561,43 +561,60 @@ server <- function(input, output, session) {
           })
           # Filter permutations -----------------------------------------------------
 
-          # output$filterperms <- renderTable({
-          #
-          #   filters <- meta$mainFile %>%
-          #     dplyr::filter(col_type == "Filter") %>%
-          #     pull(col_name)
-          #
-          #   total_info<-data.frame(filters)
-          #
-          #   for (filter in all_of(filters)) {
-          #     info <- data$mainFile %>%
-          #       select(filter) %>%
-          #       distinct()
-          #
-          #     total_info <- total_info %>%  merge(info)}
-          #
-          #
-          #   total_info <- total_info %>%
-          #     select(-filters) %>%
-          #     distinct()
-          #
-          #   #Get list of publication-specific filters
-          #   publication_filters <- meta$mainFile %>%
-          #     filter(col_type == "Filter") %>%
-          #     select(col_name) %>%
-          #     pull(col_name)
-          #
-          #   #Get filter group combos for publication-specific filters
-          #   distinct_filter_groups <- data$mainFile %>%
-          #     select(all_of(publication_filters)) %>%
-          #     distinct() %>% mutate(flag=1)
-          #
-          #   total_info %>% left_join(distinct_filter_groups) %>%
-          #     filter(is.na(flag)) %>%
-          #     select(-flag)
-          #
-          # })
-          #
+
+
+          output$downloadFilterPerms <- downloadHandler(
+            filename = function() {
+              paste("missing_filter_combinations_", basename(inputData$name), ".csv", sep = "")
```

Comment:
I think we can drop the ".csv" here as it makes it '.csv.csv'

Reply:
Yeah that % line would make cases like this a lot clearer! Will look at adding that in now - thanks :)